jupyter_notebooks/Book_4_Mountain_Wave_Moving_Mesh.ipynb

```diff
@@ -0,0 +1,508 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3ec4b5fe",
+   "metadata": {},
+   "source": [
+    "# Notebook 3: Mountain Wave Test Case"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb79138f",
+   "metadata": {},
+   "source": [
+    "This notebook will go through an example of a hydrostatic flow over a mountain. This test case is outlined in the paper: Melvin, T., Dubal, M., Wood, N., Staniforth, A., & Zerroukat, M. (2010). An inherently mass‐conserving iterative semi‐implicit semi‐Lagrangian discretization of the non‐hydrostatic vertical‐slice equations. Quarterly Journal of the Royal Meteorological Society: A journal of the atmospheric sciences, applied meteorology and physical oceanography, 136(648), 799-814."
```

Comment:
Again we can probably shorten this citation and give the link:  https://doi.org/10.1002/qj.603